indexing-service/src/test/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisorStateTest.java

```diff
@@ -2630,21 +2703,32 @@ private static DataSchema getDataSchema()
                      .build();
   }
 
-  private static SeekableStreamSupervisorIOConfig getIOConfig()
+  private static SeekableStreamSupervisorIOConfig getSupervisorIOConfig()
```

Comment:
Nit: Please use the prefix `create` instead of `get` for these new methods (you can leave the existing method names unchanged) since they are creating new objects return than getting some field that is already present in this instance.